src/Microsoft.PowerShell.Commands.Utility/commands/utility/Measure-Object.cs

```diff
@@ -266,6 +282,25 @@ public MeasureObjectCommand()
         #endregion Common parameters in both sets
 
         /// <summary>
+        /// Set to true is StdDeviation is to be returned
+        /// </summary>
+        /// <value></value>
+        [Parameter(ParameterSetName = GenericParameterSet)]
+        public SwitchParameter StdDeviation
+        {
+            get
+            {
+                return _measureStdDeviation;
+            }
+            set
+            {
+                _measureStdDeviation = value;
```

Comment:
stddeviation requires _measureAverage, so I think we should set _measureAverage = true.  If _measureAverage = false, perhaps we should error.

Reply:
Makes sense. Where should it throw, in `CreateGenericMeasureInfo`?

Reply:
Yes, I think that's the right place.

Reply:
Thinking about this, it seems that `-Average:$false -StdDeviation` should not be a valid combination and perhaps a better way to solve this is to make them different parameter sets.  The problem here is that depending on which one is set first, you may get unintended behavior `-StdDeviation -Average:$false` since the code here overwrites `_measureAverage`.

Reply:
We need new flag to process a sum. We can set it in BeginProcessing.